app/instance-initializers/bugsnag.js

```diff
@@ -1,7 +1,8 @@
 import Ember  from 'ember';
 import config from '../config/environment';
 import { getContext } from 'ember-cli-bugsnag/utils/errors';
-import { getMetaData } from '../utils/bugsnag';
+import { getMetaData } from '../utils/bugsnag-get-metadata';
```

Comment:
@binhums would you be opposed to just leaving `/utils/bugsnag` as-is (not renaming) so we can get this merged in?